ibis-server/resources/function_list/bigquery.csv

```diff
@@ -0,0 +1,101 @@
+scalar,ZEROIFNULL,FLOAT64,"Returns zero if the input is NULL."
+scalar,FORMAT,STRING,"Formats values into a string."
+scalar,LPAD,STRING,"Pads a string on the left to a certain length."
+scalar,RPAD,STRING,"Pads a string on the right to a certain length."
+scalar,LEFT,STRING,"Returns a substring from the beginning of a string."
+scalar,RIGHT,STRING,"Returns a substring from the end of a string."
+scalar,STARTS_WITH,BOOL,"Returns TRUE if the string starts with the specified prefix."
+scalar,ENDS_WITH,BOOL,"Returns TRUE if the string ends with the specified suffix."
+scalar,ARRAY_LENGTH,INT64,"Returns the length of an array."
+scalar,ARRAY_REVERSE,ARRAY,"Reverses the elements in an array."
+scalar,ARRAY_CONCAT,ARRAY,"Concatenates multiple arrays into one."
+scalar,ARRAY_TO_STRING,STRING,"Converts an array to a single string."
+scalar,GENERATE_ARRAY,ARRAY,"Generates an array of values in a range."
+scalar,GENERATE_DATE_ARRAY,ARRAY,"Generates an array of dates in a range."
+scalar,PARSE_TIMESTAMP,TIMESTAMP,"Parses a timestamp from a string."
+scalar,STRING_TO_ARRAY,ARRAY,"Splits a string into an array of substrings."
+scalar,SAFE_DIVIDE,FLOAT64,"Divides two numbers, returning NULL if the divisor is zero."
+scalar,SAFE_MULTIPLY,FLOAT64,"Multiplies two numbers, returning NULL if an overflow occurs."
+scalar,SAFE_ADD,FLOAT64,"Adds two numbers, returning NULL if an overflow occurs."
+scalar,SAFE_SUBTRACT,FLOAT64,"Subtracts two numbers, returning NULL if an overflow occurs."
+scalar,ABS,FLOAT64,"Returns the absolute value of a number."
+scalar,CEIL,INT64,"Rounds up to the nearest integer."
+scalar,FLOOR,INT64,"Rounds down to the nearest integer."
+scalar,ROUND,FLOAT64,"Rounds to the specified number of decimal places."
+scalar,TRUNC,FLOAT64,"Truncates to the specified number of decimal places."
+scalar,POW,FLOAT64,"Returns a number raised to a power."
+scalar,SQRT,FLOAT64,"Returns the square root of a number."
+scalar,LOG,FLOAT64,"Returns the natural logarithm of a number."
+scalar,LOG10,FLOAT64,"Returns the base-10 logarithm of a number."
+scalar,CONCAT,STRING,"Concatenates two or more strings."
+scalar,LOWER,STRING,"Converts a string to lowercase."
+scalar,UPPER,STRING,"Converts a string to uppercase."
+scalar,TRIM,STRING,"Removes leading and trailing whitespace."
+scalar,LTRIM,STRING,"Removes leading whitespace."
+scalar,RTRIM,STRING,"Removes trailing whitespace."
+scalar,LENGTH,INT64,"Returns the length of a string."
+scalar,REGEXP_CONTAINS,BOOL,"Returns TRUE if the string contains a match for the regex."
+scalar,REGEXP_EXTRACT,STRING,"Extracts the first match of the regex from the string."
+scalar,REGEXP_REPLACE,STRING,"Replaces all matches of the regex with a replacement string."
+scalar,SUBSTR,STRING,"Returns a substring."
+scalar,CAST,ANY,"Converts a value to a different data type."
+scalar,SAFE_CAST,ANY,"Converts a value to a different data type, returning NULL on error."
+scalar,CURRENT_TIMESTAMP,TIMESTAMP,"Returns the current timestamp."
```

Comment:
```suggestion
```
`CURRENT_TIMESTAMP` should be banned in our case.

Reply:
It should be banned in the AI service. Not here. It's an exception.

Reply:
No, Wren AI service will get the list as their white list. So, we can't add this function to the list.

Reply:
Should this function be removed only in BigQuery or all data source?

Reply:
I think we should remove all of them before we make sure they work well.